login.html

```diff
@@ -1,11 +1,46 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ko">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+    <link rel="stylesheet" href="/css/reset.css">
+    <link rel="stylesheet" href="/css/variables.css">
+    <link rel="stylesheet" href="css/auth/auth.css">
+    <title>판다마켓 : 로그인</title>
 </head>
 <body>
-    
+    <form class="form">
+        <a class="form__logo-link" href="/">
+            <img class="form__logo" src="images/logo/logo.png" alt="판다마켓 메인으로 이동">
+        </a>
+        <label class="form__label" for="email">이메일</label>
+        <input class="form__input" id="email" type="email" name="email" placeholder="이메일을 입력해주세요.">
+        <label class="form__label" for="password">비밀번호</label> 
+        <div class="form__password-input">
+            <input class="form__input" id="password" type="password" name="password" placeholder="비밀번호를 입력해주세요.">
+			<button class="form__password-toggle" type="button"></button>
+        </div>
+        <button class="form__button" type="submit">로그인</button>
```

Comment:
💬 여담
button 태그의 type 속성을 명시하지 않으면 submit을 기본값으로 가지게 됩니다~
이러한 type 속성은 button 태그에 중요한 요소이므로 명시적으로 작성해주는 것이 좋습니다.

```suggestion
        <button type="submit" class="sign-up-button">회원가입</button>
```

+type이 submit인 button의 경우 form에서 제출 이벤트가 발생하면, 해당 요소의 onclick 이벤트가 실행되게 됩니다.
이를 더 풀어서 말하자면 form 태그안의 input, button 과 같은 상호작용 요소에서 submit 이벤트, enter key 입력이 발생하면 해당 form 에서의 submit 이벤트가 실행된다는 것입니다.
자세한 것은 추후 학습 하시면서 배우실 것이니 우선은 button 태그의 type 속성을 명시적으로 작성하는 것이 좋다는 것만 기억해주시면 좋겠습니다~

[mdn button](https://developer.mozilla.org/ko/docs/Web/HTML/Element/button#type)